lego-webapp/pages/events/src/EventEditor/renderPools.tsx

```diff
@@ -64,6 +64,7 @@ const renderPools = ({ fields, startTime, eventStatusType }: poolProps) => (
           component={DatePicker.Field}
         />
         <Field
+          disabled={fields.value[index].registrations?.length > 0}
```

Comment:
What about if they want to update a pool which has opened but gotten zero registrations?

Reply:
I agree that that is probably not an issue, my only concern about it is that it might seem like a confusing requirement and that it might be easier to enforce it based on whether or not the pool has opened.